sdk/core/core-client-rest/src/sendRequest.ts

```diff
@@ -174,9 +173,7 @@ function getRequestBody(body?: unknown, contentType: string = ""): RequestBody {
 
   if (ArrayBuffer.isView(body)) {
     if (body instanceof Uint8Array) {
-      return firstType === "application/octet-stream"
-        ? { body }
-        : { body: binaryArrayToString(body) };
+      return { body };
     } else {
       return { body: JSON.stringify(body) };
     }
```

Comment:
nitpick: this could be simplified to `return { body: body instanceof Uint8Array ? body : JSON.stingify(body) };`